web/api/Makefile

```diff
@@ -50,6 +50,7 @@ build: clean target_dirs
 	npm pack && \
 	mkdir -p dist && \
 	mv -f codechecker-api-*.tgz dist && \
+	git add dist && \
```

Comment:
Why are we adding all the tarball gz files to git in the `make build` command itself? Since we have removed `dist ` from `.gitignore` these files will get added during manual `git add` operation

Reply:
The problem at here is that for node it will generate a local package with a new name (version number is in the file name), so this file will be marked as Untracked file by git. And I want to avoid that someone will forget to add this new zip file to the git. And I think this is the common use case when someone modify these files he will add it to the git at the end.